wasm-mappings/CONTRIBUTING.md

```diff
@@ -0,0 +1,87 @@
+# Contributing to `source-map-mappings`
+
+Hi! We'd love to have your contributions! If you want help or mentorship, reach
+out to us in a GitHub issue, or ping `fitzgen` in [#rust on irc.mozilla.org](irc://irc.mozilla.org#rust)
```

Comment:
Is this irc channel still valid?

Reply:
irc server is down, I removed it with the new matrix channel.